code/modules/research/designs/medical_designs.dm

```diff
@@ -218,7 +229,7 @@
 	req_tech = list("materials" = 6, "programming" = 4, "biotech" = 7, "syndicate" = 4)
 	build_type = PROTOLATHE | MECHFAB
 	construction_time = 60
-	materials = list(MAT_METAL = 200, MAT_GLASS = 200, MAT_GOLD = 500, MAT_URANIUM = 1000, MAT_DIAMOND = 2000)
+	materials = list(MAT_METAL = 2000, MAT_GLASS = 2000, MAT_GOLD = 6000, MAT_URANIUM = 2000, MAT_DIAMOND = 4000)
```

Comment:
It's practically nothing with upgraded machines.

And even then, it's one of the best implants and it just got buffed too.